src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -57,16 +59,16 @@ class HuiEntityButtonCard extends hassLocalizeLitMixin(LitElement)
   }
 
   protected shouldUpdate(changedProps: PropertyValues): boolean {
-    const newHass = changedProps.get("hass") as any;
-    if (!changedProps.has("_config") && newHass) {
+    const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
```

Comment:
move this below the if-check so you are not doing work that you won't use.

Reply:
Oh wait a second. this is inside `shouldUpdate`, so in this case `changedProps` are the NEW values. My bad.